kalite/facility/templates/facility/login.html

```diff
@@ -25,7 +25,8 @@ <h3 class="no-margin-top">{% trans "Log in" %}</h3>
         </form>
         <div id="info_block">
             {% if not restricted %}
-                {% blocktrans %}If you don't have a login, please <a href="{{ sign_up_url }}">sign up</a>.{% endblocktrans %}
+                {% url 'facility_user_signup' as signup_url %}
+                {% blocktrans %}If you don't have a login, please <a href="{{ signup_url }}">sign up</a>.{% endblocktrans %}
```

Comment:
Yeah, it's a "feature" of blocktrans that you cannot put other block tags inside it.. just simple variable substitution. Not even template filters are allowed.